numpy/core/src/private/npy_import.h

```diff
@@ -29,4 +29,18 @@ npy_cache_import(const char *module, const char *attr, PyObject **cache)
     }
 }
 
+NPY_INLINE static PyObject *
+npy_import(const char *module, const char *attr)
```

Comment:
Why was this function added?

Reply:
Oh... I thought it was used in `_multifield_view_to_copy`, but I must have removed that. Yes, I think it was related to the config variabel that I removed.

I'll make a followup to remove this.